doc/manuals/user/filtering.md

```diff
@@ -10,26 +10,25 @@ As a general rule, filters used in standard operation use a scope
-      <queryContextRequest>                                                                                                         "entities": [
-        <entityIdList>                                                                                                              {
-          <entityId type="myEntType" isPattern="true">                                                                                "type": "myEntityType",
-            <id>.*</id>                                                                                                               "isPattern": "true",
-          </entityId>                                                                                                                 "id": ".*"
-        </entityIdList>                                                                                                             }
-        <attributeList>                                                                                                             ],
-        </attributeList>                                                                                                            "restriction": {
-        <restriction>                                                                                                                 "scopes": [
-          <scope>                                                                                                                       {
-            <operationScope>                                                                                                              "type" : "FIWARE::Filter::foobar",
-              <scopeType>FIWARE::Filter::foobar</scopeType>                                                                               "value" : ...
-              <scopeValue>...</scopeValue>                                                                                              }
-            </operationScope>                                                                                                         ]
-          </scope>                                                                                                                  }
-        </restriction>                                                                                                            }
-      </queryContextRequest>                                                                                                      EOF
-      EOF                                                                                                                     
+      (curl localhost:1026/v1/queryContext -s -S --header 'Content-Type: application/json' --header 'Accept: application/json' -d @- | python -mjson.tool) <<EOF
+      {
+	"entities": [
+        {
+	    type": "myEntityType",
+            "isPattern": "true",
+	    "id": ".*"
+        }
```

Comment:
Sometimes italics are shown in the diff, but if you look to the rendered version of the file (in this case https://github.com/telefonicaid/fiware-orion/blob/hardening/RemovedXML/doc/manuals/user/filtering.md) and it's ok, then it is not actually a problem.